creepy/subprocess/pypen.py

```diff
@@ -141,6 +146,23 @@ def detach(self):
         except (OSError, FileNotFoundError, AttributeError):
             pass
 
+    @lru_cache(maxsize=1)
```

Comment:
This cache is useless. Normally it won't be called twice with the same `self`. If you create another instance of the same Pypen, it will compile the same interface again.

Reply:
Moreover, maxsize=1 means that it will keep just one compiled interface for a single Pypen instance.